api/v1alpha1/nicclusterpolicy_types.go

```diff
@@ -291,6 +291,8 @@ type AppliedState struct {
 	Name string `json:"name"`
 	// +kubebuilder:validation:Enum={"ready", "notReady", "ignore", "error"}
 	State State `json:"state"`
+	// Error string in case the state is error
+	Error string `json:"error,omitempty"`
```

Comment:
can we call this "message"  ? 

// message is a human readable message indicating details about why the state is in this condition.

similar to what we have in pod status

Reply:
Done